docs/source/config.rst

```diff
@@ -112,7 +112,6 @@ The answers to these questions are provided by the various
 :class:`~traitlets.config.Configurable` subclasses that an
 application uses. Let's look at how this would work for a simple configurable
 subclass::
-
```

Comment:
I think the right fix here is to remove the `::`, not remove the blank line. The `::` is redundant with the `.. code-block:` below.